src/Workspaces/VisualBasic/Portable/Classification/Classifiers/IdentifierNameSyntaxClassifier.vb

```diff
@@ -0,0 +1,34 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Threading
+Imports Microsoft.CodeAnalysis.Classification
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+Imports Microsoft.CodeAnalysis.VisualBasic.Extensions.ContextQuery
+Imports Microsoft.CodeAnalysis.VisualBasic.Symbols
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.Classification.Classifiers
+    Friend Class IdentifierNameSyntaxClassifier
+        Inherits AbstractSyntaxClassifier
+
+        Private Const AwaitText = "Await"
+
+        Public Overrides ReadOnly Property SyntaxNodeTypes As IEnumerable(Of Type)
+            Get
+                Return {GetType(IdentifierNameSyntax)}
+            End Get
+        End Property
+
+        Public Overrides Function ClassifyNode(syntax As SyntaxNode, semanticModel As SemanticModel, cancellationToken As CancellationToken) As IEnumerable(Of ClassifiedSpan)
+            Dim identifierName = DirectCast(syntax, IdentifierNameSyntax)
+            Dim identifier = identifierName.Identifier
+            If StringComparer.OrdinalIgnoreCase.Equals(identifier.ValueText, AwaitText) Then
```

Comment:
I seem to recall there being some helper class for comparing identifiers in VB, but I can't find it. Maybe my memory isn't what it used to be? @jasonmalinowski, do you know?